### PR TITLE
move app_manager/models to its own dir

### PR DESCRIPTION
**Pull request by dannyroberts**
in a merge-friendly way

### CODE REVIEW

**Comment by dimagimon** [comment]
Could not review pull request. It may be too large, or contain no reviewable changes.

File: corehq/apps/app_manager/models/models.py [hunk 1]
```diff
@@ -82,7 +82,7 @@
 from corehq.apps.app_manager.xform import XForm, parse_xml as _parse_xml, \
     validate_xform
 from corehq.apps.app_manager.templatetags.xforms_extras import trans
-from .exceptions import (
+from ..exceptions import (
```
Comment (dimagimon) [inline corehq/apps/app_manager/models/models.py]: F401 'NoMatchingFilterException' imported but unused

**Comment by czue** [comment]
(not mergeable)

**Comment by dannyroberts** [comment]
The process for a merge-friendly move went something like this:

```bash
scripts/git-good-move corehq/apps/app_manager/ corehq/apps/app_manager/models/
git reset
git add corehq/apps/app_manager/models/models.py
# the one file we care about stays staged
rm -rf corehq/apps/app_manager/models/
git checkout -- .
touch corehq/apps/app_manager/models/__init__.py
```

**Comment by dannyroberts** [comment]
@czue yeah when anything gets merged into app_manager/models.py on master, I still have to manually `git merge` locally and push; but it's an automatic merge with no conflicts :)

**Comment by dannyroberts** [comment]
@orangejenny I'd advocate merging this and then using it as a starting point. You're going to have a much easier time splitting models out once we're in a stable models-as-dir situation on master.

**Comment by dannyroberts** [comment]
@orangejenny 

Hmm, looks like we'll have to apply some of the fixes you made on your other branch to this one as well.

I'd still advocate applying them here, getting the tests to pass, and merging this, and then splitting in a separate PR. What do you think?

**Comment by orangejenny** [comment]
I thought I'd encountered problems with having a models/models.py, but might be mis-remembering. Sure, I'm fine with creating the models directory in this PR before starting to split off smaller files.

**Comment by proteusvacuum** [comment]
@dannyroberts @orangejenny are you still thinking of working on this?

**Pull request closed without merging**